deps.bzl

```diff
@@ -40,6 +40,14 @@ def prysm_deps():
         sum = "h1:UoveltGrhghAA7ePc+e+QYDHXrBps2PqFZiHkGR/xK8=",
         version = "v0.0.1-2020.1.4",
     )
+    go_repository(
+        name = "com_github_ferranbt_fastssz",
+        importpath = "github.com/ferranbt/fastssz",
+        nofuzz = True,
+        replace = "github.com/kasey/fastssz",
+        sum = "h1:Y5APFBP61jxeJTpuN2kme22OxF3+mOqQ9kmpyhKyiOc=",
+        version = "v0.0.0-20210624164955-4e94943eb307",
+    )
```

Comment:
Curious, is there a PR open to merge these changes upstream? Could you link to it in this PR?

Reply:
This builds on top of Raul's PR https://github.com/ferranbt/fastssz/pull/46

I wanted that PR to be merged first so that my diff wouldn't include his changes. I can either set up a merge targeting his branch or just go ahead and set up the PR with all the changes. I marked this PR WIP for now.